public/stylesheets/main.css

```diff
@@ -79,3 +79,16 @@
   justify-content: space-between;
   width: 100%;
 }
+
+.change-rating-h2 {
+  font-size: large;
```

Comment:
It's not a good practice to use absolute size values for `font-size`. Better use some value with units, for example, `1.5em` or similar.
https://developer.mozilla.org/en-US/docs/Web/CSS/font-size